src/core/trtserver.cc

```diff
@@ -196,6 +289,89 @@ TRTSERVER_ProtobufSerialize(
   return nullptr;  // Success
 }
 
+//
+// TRTSERVER_MemoryAllocator
+//
+TRTSERVER_Error*
+TRTSERVER_MemoryAllocatorNew(
+    TRTSERVER_MemoryAllocator** allocator, TRTSERVER_MemoryAllocFn_t alloc_fn,
+    TRTSERVER_MemoryDeleteFn_t delete_fn)
+{
+  return nullptr;  // Success
```

Comment:
Is this supposed to be implemented?

Reply:
Not yet... it is going to require some provider changes that I will make in a future change. But I will add a note in the code